pom.xml

```diff
@@ -269,12 +269,7 @@
                 <groupId>org.bouncycastle</groupId>
                 <artifactId>bcpkix-jdk15on</artifactId>
                 <version>${bouncycastle.version}</version>
-            </dependency>
-            <!-- must be direct dependency to be embedded -->
-            <dependency>
-                <artifactId>bcprov-jdk15on</artifactId>
-                <groupId>org.bouncycastle</groupId>
-                <version>${bouncycastle.version}</version>
+                <scope>provided</scope>
```

Comment:
So `bcpkix-jdk15on` we would always embed, and `bcprov-jdk15on` users only install if they need it?

Reply:
No, both need to be installed (we don't embed anything). Please compare with the documentation.

Reply:
With current version 2.6.2 we embed both bcpkix and bcprov, don't we break backwards compatibility if this needs to be installed in a different way now?